baremetal_network_provisioning/db/bm_nw_provision_models.py

```diff
@@ -47,3 +47,56 @@ class HPIronicSwitchPortMapping(model_base.BASEV2):
     access_type = sa.Column(sa.String(36))
     segmentation_id = sa.Column(sa.Integer)
     host_id = sa.Column(sa.String(36))
+
+
+class BNPPhysicalSwitchPort(model_base.BASEV2, models_v2.HasId):
+    """Define physical switch port properties."""
+    __tablename__ = "bnp_physical_switch_ports"
+    switch_id = sa.Column(sa.String(255), nullable=False)
+    interface_name = sa.Column(sa.String(255), nullable=False)
+    ifindex = sa.Column(sa.String(255), nullable=False)
+    port_status = sa.Column(sa.String(16), nullable=False)
+    sa.PrimaryKeyConstraint('id')
+    sa.UniqueConstraint('switch_id', 'interface_name')
+
+
+class BNPPhysicalSwitch(model_base.BASEV2, models_v2.HasId):
+    """Define physical switch properties."""
+    __tablename__ = "bnp_physical_switches"
+    ip_address = sa.Column(sa.String(64), nullable=False)
```

Comment:
In case of V1/V2c write community needs to be stored.. I think it got missed here